### PR TITLE
days seeded data up

### PR DESCRIPTION
**Pull request by madhalle**
Data seeded and tables set up in DB. Db switched from sqllite to PSQL

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -1,11 +1,24 @@
-from flask import Flask
+from flask import Flask, jsonify
 from flask_sqlalchemy import SQLAlchemy
 from flask_cors import CORS, cross_origin
-
+import os
+#create app method
 app = Flask(__name__)
-cors = CORS(app)
+cors = CORS(app, support_credentials=True)
```
Comment (madhalle) [inline app/__init__.py:7]: trying to work in CORS. This is breaking the add_day endpoint currently.

File: app/models.py [hunk 2]
```diff
@@ -1,19 +1,12 @@
...
-manager.add_command('db', MigrateCommand)
 class Day(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     temperature = db.Column(db.Float, nullable=False)
     date = db.Column(db.String(20), nullable=False)
-    high_risk = db.Column(db.Boolean(), nullable=True)
+    high_risk = db.Column(db.Boolean, nullable=True)
 
-    def __init__(self, temperature, date):
+    def __init__(self, temperature, date, high_risk):
```
Comment (madhalle) [inline app/models.py:9]: now that I'm seeing this I dont think we need high risk here

File: app/routes.py [hunk 3]
```diff
@@ -25,12 +26,15 @@ def returnAll():
 
 
 @app.route('/add_day', methods=['POST'])
+@cross_origin(supports_credentials=True)
```
Comment (madhalle) [inline app/routes.py:29]: CORS decorator

File: manage.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
+from app import app, db
```
Comment (madhalle) [inline manage.py:1]: Added a manage file to run some built up seeder commands

File: migrations/env.py [hunk 5]
```diff
@@ -0,0 +1,96 @@
+from __future__ import with_statement
```
Comment (madhalle) [inline migrations/env.py:1]: autogenerated with migrations

File: run.py [hunk 6]
```diff
@@ -1,4 +1,5 @@
 from app import app
 
 if __name__ == "__main__":
-    app.run(debug=True)
\ No newline at end of file
+    # app.run(debug=True)
```
Comment (madhalle) [inline run.py:4]: CORS permissions

File: app/__init__.py (hunk 1 above)
Comment (whitneykidd) [reply app/__init__.py:7]: Does this mean it's working now with these changes?

File: app/models.py (hunk 2 above)
Comment (whitneykidd) [reply app/models.py]: yeah, that makes sense to add that attribute ourselves after we do some calculations

**Pull request merged**